common-rest/src/main/java/org/uniprot/api/rest/openapi/OpenApiConstants.java

```diff
@@ -0,0 +1,680 @@
+                    + "The databases are categorized for easy user perusal and understanding of how the "
+                    + "different databases relate to both UniProtKB and to each other";
+    public static final String QUERY_CROSSREF_DESCRIPTION =
+            "Criteria to search cross-reference databases. "
+                    + QUERY_ADVANCED
+                    + " <a href='https://rest.uniprot.org/configure/uniprotkb/search-fields'>List of valid search fields</a>";
+    public static final String QUERY_CROSSREF_EXAMPLE = "Ensembl";
+    public static final String SORT_CROSSREF_DESCRIPTION =
+            SORT_DESCRIPTION
+                    + " <a href='https://rest.uniprot.org/configure/uniprotkb/sort'>List of valid sort fields</a>";
+    public static final String SORT_CROSSREF_EXAMPLE = "id desc";
+    public static final String FIELDS_CROSSREF_DESCRIPTION =
+            FIELDS_DESCRIPTION
+                    + "  <a href='https://rest.uniprot.org/configure/uniprotkb/result-fields'>List of valid fields</a>";
+    public static final String FIELDS_CROSSREF_EXAMPLE = "id,name,abbrev,category";
+    public static final String ID_CROSSREF_DESCRIPTION =
+            "Unique identifier for the cross-reference database entry";
+    public static final String ID_CROSSREF_EXAMPLE = "DB-0244";
+    public static final String ID_CROSSREF_OPERATION =
+            "Get cross-reference database entry by a single accession.";
+    public static final String SEARCH_CROSSREF_OPERATION =
+            "Retrieve cross-reference database entries by a search query.";
+    public static final String STREAM_CROSSREF_OPERATION =
+            "Download cross-reference database entries retrieved by a search query.";
+
+    // GeneCentric
+    public static final String TAG_GENECENTRIC = "GeneCentric";
+    public static final String TAG_GENECENTRIC_DESC = "GeneCentric services of a proteome, where the set of genes and their products are grouped under a single canonical gene identifier";
+    public static final String SEARCH_GENECENTRIC_OPERATION =
+            "Retrieve GeneCentric entries by a search query.";
+    public static final String STREAM_GENECENTRIC_OPERATION =
+            "Download GeneCentric entries retrieved by a search query.";
+    public static final String UPID_GENECENTRIC_OPERATION = "Retrieve GeneCentric entries of a single Proteome ID.";
+    public static final String ID_GENECENTRIC_OPERATION =
+            "Retrieve a GeneCentric entry by a single UniProtKB accession.";
+    public static final String QUERY_GENECENTRIC_DESCRIPTION =
+            "Criteria to search GeneCentric. "
+                    + QUERY_ADVANCED
+                    + " <a href='https://rest.uniprot.org/configure/uniprotkb/search-fields'>List of valid search fields</a>";;
+    public static final String QUERY_GENECENTRIC_EXAMPLE = "gene:APP";
```

Comment:
Semicolon repeated. There are other places in this file where semicolon is repeated